proto/common.proto

```diff
@@ -6,16 +6,64 @@ message Error { string cause = 1; }
 
 message Close { string channel = 1; }
 
-message Message {
+// Central piece of data
+message DataChunk { bytes data = 1; }
+
+// This message contains data that is sent from a runner to the orchestrator
+message SendingMessage {
   string channel = 1;
   bytes data = 2;
+  uint32 localSequenceNumber = 3;
 }
 
-message Id { uint64 id = 1; }
-
-message DataChunk { bytes data = 1; }
+// This message contains data that is received by a runner from the orchestrator
+message ReceivingMessage {
+  string channel = 1;
+  bytes data = 2;
+  uint32 globalSequenceNumber = 3;
+}
 
-message StreamMessage {
-  Id id = 1;
+// StreamMessage contains the sequenceNumber used to receive the streaming data
+// This message is received by the runner from the orchestrator
+message ReceivingStreamMessage {
+  uint32 globalSequenceNumber = 1;
```

Comment:
Does order matter in protobuf? Otherwise it might make more sense to keep the same order as the `ReceivingMessage` from above, i.e., first `channel` and then `globalSequenceNumber`.